tests/console/commands/test_add.py

```diff
@@ -1506,3 +1511,29 @@ def test_add_extras_only_accepts_one_package(
             str(e.value)
             == "You can only specify one package when using the --extras option"
         )
+
+
+@pytest.mark.parametrize("command", ["foo", "foo --lock"])
```

Comment:
```suggestion
@pytest.mark.parametrize("command", ["foo", "foo --lock"])
@pytest.mark.parametrize(("locked", "expected_docker"), [(True, "4.3.1"), (False, "4.3.2")])
```

Thinking about the `poetry.locker.locked(True)`, I just came up with an idea to make the test more robust.

Reply:
Great suggestion! I just added it ;)